### PR TITLE
Add Windows hotfixes to cache into the VHD.

### PR DESCRIPTION
**Pull request by yizhang4321**
Added Wiindows hotfixes for the following versions:
v1.16.14
v1.17.7
v1.17.9
v1.18.6
v1.18.8

### CODE REVIEW

File: vhdbuilder/packer/configure-windows-vhd.ps1 [hunk 1]
```diff
@@ -115,20 +115,25 @@ function Get-FilesToCacheOnVHD {
             "https://acs-mirror.azureedge.net/kubernetes/v1.17.9-azs/windowszip/v1.17.9-azs-1int.zip",
             "https://acs-mirror.azureedge.net/kubernetes/v1.17.11-azs/windowszip/v1.17.11-azs-1int.zip",
             "https://acs-mirror.azureedge.net/kubernetes/v1.16.10-hotfix.20200817/windowszip/v1.16.10-hotfix.20200817-1int.zip",
+            "https://acs-mirror.azureedge.net/kubernetes/v1.16.10-hotfix.20200824/windowszip/v1.16.10-hotfix.20200824-1int.zip",
```
Comment (AbelHu) [inline vhdbuilder/packer/configure-windows-vhd.ps1:118]: I think that we can remove this one since 20200917 is added

Comment (AbelHu) [reply vhdbuilder/packer/configure-windows-vhd.ps1:118]: I think that we do not need to add any new package for 1.16.10 since AKS does not support it

Comment (yizhang4321) [reply vhdbuilder/packer/configure-windows-vhd.ps1]: Thanks for your review, Abel. I removed the two hotfixes for 1.16.10.

File: vhdbuilder/packer/configure-windows-vhd.ps1 [hunk 2]
```diff
@@ -122,13 +122,16 @@ function Get-FilesToCacheOnVHD {
             "https://acs-mirror.azureedge.net/kubernetes/v1.16.13-hotfix.20200824/windowszip/v1.16.13-hotfix.20200824-1int.zip",
             "https://acs-mirror.azureedge.net/kubernetes/v1.16.13-hotfix.20200917/windowszip/v1.16.13-hotfix.20200917-1int.zip",
             "https://acs-mirror.azureedge.net/kubernetes/v1.16.14/windowszip/v1.16.14-1int.zip",
+            "https://acs-mirror.azureedge.net/kubernetes/v1.16.14-hotfix.20200901/windowszip/v1.16.14-hotfix.20200901-1int.zip",
```
Comment (AbelHu) [inline vhdbuilder/packer/configure-windows-vhd.ps1:125]: AKS only supports 1.16.13 and 1.16.15.

File: vhdbuilder/packer/configure-windows-vhd.ps1 [hunk 3]
```diff
@@ -122,13 +122,16 @@ function Get-FilesToCacheOnVHD {
...
             "https://acs-mirror.azureedge.net/kubernetes/v1.16.15-hotfix.20200903/windowszip/v1.16.15-hotfix.20200903-1int.zip",
             "https://acs-mirror.azureedge.net/kubernetes/v1.17.7-hotfix.20200817/windowszip/v1.17.7-hotfix.20200817-1int.zip",
+            "https://acs-mirror.azureedge.net/kubernetes/v1.17.7-hotfix.20200917/windowszip/v1.17.7-hotfix.20200917-1int.zip",
```
Comment (AbelHu) [inline vhdbuilder/packer/configure-windows-vhd.ps1:128]: AKS only supports 1.17.11 and 1.17.13

File: vhdbuilder/packer/configure-windows-vhd.ps1 [hunk 4]
```diff
@@ -137,8 +140,10 @@ function Get-FilesToCacheOnVHD {
             "https://acs-mirror.azureedge.net/kubernetes/v1.18.5/windowszip/v1.18.5-1int.zip",
             "https://acs-mirror.azureedge.net/kubernetes/v1.18.6/windowszip/v1.18.6-1int.zip",
             "https://acs-mirror.azureedge.net/kubernetes/v1.18.6-hotfix.20200723/windowszip/v1.18.6-hotfix.20200723-1int.zip",
+            "https://acs-mirror.azureedge.net/kubernetes/v1.18.6-hotfix.20200917/windowszip/v1.18.6-hotfix.20200917-1int.zip",
```
Comment (AbelHu) [inline vhdbuilder/packer/configure-windows-vhd.ps1:143]: It seems like that there is no new hotfixes for supported k8s versions.

**Comment by yizhang4321** [comment]
Abandoning as discussed with Abel. No new hotfix to add for this release.

**Pull request closed without merging**